cmd/operator-sdk/run/local.go

```diff
@@ -179,6 +188,10 @@ func setupOperatorEnv(kubeconfig, namespace string) error {
 	return nil
 }
 
+// todo: remove when the legacy layout is no longer supported
+// Deprecated: Used just for the legacy layout
+// --
+// buildLocal will build the project
```

Comment:
This function can be generically used to build Go programs. We can continue using it for KB projects.

Reply:
Solved.